parts/k8s/addons/1.16/kubernetesmasteraddons-azure-npm-daemonset.yaml

```diff
@@ -79,8 +79,10 @@ spec:
             privileged: true
           resources:
             requests:
+              cpu: {{ContainerCPULimits "azure-npm-daemonset"}}
```

Comment:
These values are swapped. ContainerCPULimits needs to be under "limits:", and ContainerCPUReqs needs to be under "requests:".